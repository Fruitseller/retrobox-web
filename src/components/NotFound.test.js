import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  shallow(<NotFound />);
});

it('renders correctly', () => {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
