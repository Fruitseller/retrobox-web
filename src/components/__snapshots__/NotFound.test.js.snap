// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2>
  Not Found
</h2>
`;
